build.psm1

```diff
@@ -861,8 +890,6 @@ function Start-PSBootstrap {
                 }
             }
 
-        } elseif ($IsWindows) {
-            Write-Warning "Start-PSBootstrap cannot be run in Core PowerShell on Windows (need Invoke-WebRequest!)"
```

Comment:
This warning was old, as `Invoke-WebRequest` is now available in latest packages.